frontend/src/components/card/moment/audioCard.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+
+import { IonCol, IonButton, IonIcon, IonGrid, IonRow } from '@ionic/react'
+
+import { playCircle, pauseCircle } from 'ionicons/icons'
+
+export interface AudioVariant {
+  variant: 'audio'
+  audio: string
+}
+export const AudioVariantCard: React.FC<AudioVariant> = props => {
+  const [play, setPlay] = React.useState(true)
+  const clickButton = () => {
+    setPlay(e => !e)
+  }
+  return (
+    <IonGrid>
+      <IonRow className="ion-align-items-center">
+        <IonCol></IonCol>
+        <IonCol>
+          <IonButton size="large" fill="clear" onClick={clickButton}>
+            <IonIcon
+              slot="icon-only"
+              color="primary"
+              icon={play ? playCircle : pauseCircle}
+            />
+          </IonButton>
+        </IonCol>
+        <IonCol></IonCol>
```

Comment:
Why?

Reply:
I found no other solution to center the content. As you can see there are 3 Cols, two are empty and only exist to center the second col.
https://github.com/oseven-development/petrakip-app/blob/ea6d9f444612be0f70239739c55990edd93c0968/frontend/src/components/card/moment/audioCard.tsx#L19

and 

https://github.com/oseven-development/petrakip-app/blob/ea6d9f444612be0f70239739c55990edd93c0968/frontend/src/components/card/moment/audioCard.tsx#L29

Reply:
This can't be right!! don't use a grid? Have you looked here? 
https://ionicframework.com/docs/layout/css-utilities
And you always can add inline styles with `flex` or `margin`

Reply:
Yae i will search and try to kick the Cols out